data_pipeline/utils/__init__.py

```diff
@@ -0,0 +1,57 @@
+
+
+def update_deployment_env_placeholder(
```

Comment:
Would probably suggest to move them to a dedicated module. Just "utils" feels slightly to generic. It does seem to be related to configuration.